src/addBusinessDays/index.ts

```diff
@@ -27,19 +31,31 @@ export default function addBusinessDays(
   dirtyAmount: number,
   dirtyOptions?: {
     businessDays?: number[]
+    exceptions?: Record<string, boolean>
   }
 ): Date {
   requiredArgs(2, arguments)
   const options = dirtyOptions || {}
+  const exceptions = options.exceptions || {}
   const businessDays =
     options.businessDays == null
       ? [1, 2, 3, 4, 5]
-      : options.businessDays.map(toInteger)
+      : options.businessDays.filter((number) => number < 7).map(toInteger)
```

Comment:
@chr1sjf0x @khalidwilliams  Looks like I'm too late on this comment, but I'm curious about the use of these filters on `businessDays`  - am I correct in assuming this is to protect against an invalid `day` index? If so I would think we'd want to deal with this by throwing an error explicitly with a helpful message rather than silently filtering them away?

Reply:
Got it, yeah looks like the lib uses the convention of throwing a `RangeError` in the event of invalid input - some examples to check out: https://github.com/date-fns/date-fns/search?q=RangeError